.github/workflows/checks.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     - name: Install dependencies
       run: |
         echo '::group::Output of "conda install" commands'
-        conda install --quiet --yes pip=21.1 wheel setuptools
+        conda install --quiet --yes pip=21.1 wheel setuptools pandoc
```

Comment:
I think it is fine to add `pandoc` here for the time being, but eventually we should have a more explicit way to collect all non-`pip`-installable dev and/or optional dependencies (e.g. an `environment-dev.yml` file for Conda that internally calls `requirements-dev.txt` in the `pip` section).

Reply:
If that's the "right" way to do it, then sure. Should go with some README developer docs since this is pretty obscure.